plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/TestPinotSessionProperties.java

```diff
@@ -44,4 +44,18 @@ public void testConnectionTimeoutParsedProperly()
                 .build();
         assertThat(PinotSessionProperties.getConnectionTimeout(session)).isEqualTo(new Duration(0.25, TimeUnit.MINUTES));
     }
+
```

Comment:
We can define the variable like below and use it in assertion method. 
```
        Session queryOptions = Session.builder(getQueryRunner().getDefaultSession())
                .setCatalogSessionProperty("pinot", "query_options", "...")
                .build();
...
```
Please refer to BasePinotConnectorSmokeTest.testAggregationPushdown.

Reply:
This is connector config. I tried session catalog property. It complains its not a recognized session property.

Reply:
I can use the above code. How did you try?

Reply:
Exacty what you have there. It compiles but since that is not catalog session property, I see a runtime exception.
`Session queryOptions = Session.builder(getQueryRunner().getDefaultSession())
                .setCatalogSessionProperty("pinot", "query_options", "skipUpsert=true")
                .build();
        assertThat(query(queryOptions,"SELECT city, \"sum(long_number)\" FROM" +
                         " \"SET skipUpsert = 'true';" +
                         " SET numReplicaGroupsToQuery = '1';" +
                         " SELECT city, SUM(long_number)" +
                         "  FROM my_table" +
                         "  GROUP BY city" +
                         "  HAVING SUM(long_number) > 10000\""))
                .matches("VALUES (VARCHAR 'Los Angeles', DOUBLE '50000.0'), (VARCHAR 'New York', DOUBLE '20000.0')")
                .isFullyPushedDown();`

Reply:
The session value `skipUpsert=true` should be `skipUpsert:true`.

Reply:
Gentle reminder.

Reply:
Reminder.

Reply:
Done